app/views.py

```diff
@@ -1,3 +1,5 @@
+from constance import config
+from datetime import datetime, timedelta
```

Comment:
import this from django i think to properly do timezone etc or smt

Reply:
Ok changed to `from django.utils import timezone`